keadm/cmd/keadm/app/cmd/util/image.go

```diff
@@ -152,7 +152,7 @@ func (runtime *DockerRuntime) RemoveMQTT() error {
 		All: true,
 	}
 	options.Filters = filters.NewArgs()
-	options.Filters.Add("ancestor", constants.DefaultMosquittoImage)
+	options.Filters.Add("name", image.EdgeMQTT)
```

Comment:
Where is the key `name` and value `image.EdgeMQTT` set? :)

Reply:
i don't find the ancetor be setted any where, so i use the container name to handle it, the container name is setted when the mqtt container run, be like https://github.com/kubeedge/kubeedge/blob/d0d3c8ac461b2d1882907f4743b565e052789494/keadm/cmd/keadm/app/cmd/util/image.go#L143 and https://github.com/kubeedge/kubeedge/blob/d0d3c8ac461b2d1882907f4743b565e052789494/keadm/cmd/keadm/app/cmd/util/image.go#L377